doc/contribute/doc_guidelines.rst

```diff
@@ -0,0 +1,378 @@
+.. _doc_guidelines:
+
+Documentation Guidelines
+########################
+
+Project ACRN content is written using the `reStructuredText`_ markup
+language (``.rst`` file extension) with Sphinx extensions, and processed
+using Sphinx to create a formatted standalone website.  Developers can
+view this content either in its raw form as .rst markup files, or (with
+Sphinx installed) they can build the documentation using the Makefile
+(on Linux systems) to
+generate the HTML content. The HTML content can then be viewed using a
+web browser. This same ``.rst`` content is also fed into the
+`Project ACRN documentation`_ website.
+
+You can read details about `reStructuredText`_
+and about `Sphinx extensions`_ from their respective websites.
+
+.. _Sphinx extensions: http://www.sphinx-doc.org/en/stable/contents.html
+.. _reStructuredText: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+.. _Sphinx Inline Markup:  http://sphinx-doc.org/markup/inline.html#inline-markup
+.. _Project ACRN documentation:  http://projectacrn.github.io
+
+This document provides a quick reference for commonly used reST and
+Sphinx-defined directives and roles used to create the documentation
+you're reading.
+
+Headings
+********
+
+While reST allows use of both and overline and matching underline to
+indicate a heading, we only use an underline indicator for headings.
+
+* Document title (h1) use "#" for the underline character
+* First section heading level (h2) use "*"
+* Second section heading level (h3) use "="
+* Third section heading level (h4) use "-"
+
+Additional heading level depth is discouraged.
+
+The heading underline must be at least as long as the title it's under.
+
+For example:
+
+.. code-block:: rest
+
+   This is a title heading
+   #######################
+
+   some content goes here
+
+   First section heading
+   *********************
+
+   Second section heading too short
+   **********
```

Comment:
I think you meant to use `======` here instead (second section heading)

Reply:
no, I just meant the second section at the same level (not second level)... I'll tweak the text.